net-libs/libp8_platform/libp8_platform-2.1.0.1.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Platform support library used by libCEC and binary add-ons for Kodi"
+DESCRIPTION="This library provides platform specific support for other libraries"
+HOMEPAGE="http://libcec.pulse-eight.com/"
+COPYRIGHT="2011-2015 Pulse-Eight Limited"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/Pulse-Eight/platform/archive/p8-platform-$portVersion.tar.gz"
+CHECKSUM_SHA256="064f8d2c358895c7e0bea9ae956f8d46f3f057772cb97f2743a11d478a0f68a0"
+SOURCE_DIR="platform-p8-platform-$portVersion"
+PATCHES="libp8_platform-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libp8_platform$secondaryArchSuffix = $portVersion
+	lib:libp8_platform$secondaryArchSuffix = $portVersion
```

Comment:
```suggestion
	lib:libp8_platform$secondaryArchSuffix = 2.1.0 compat >= 2
```

Reply:
It'd be better to create a `libVersion` variable: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-libversioncompat-user-variable